Silicon/CommonSocPkg/Library/BootGuardLibCBnT/BootGuardLibrary.c

```diff
@@ -78,18 +78,23 @@ GetBootGuardInfo (
     BootGuardBootStatus  = *(UINT64 *) (UINTN) (TXT_PUBLIC_BASE + R_CPU_BOOT_GUARD_BOOTSTATUS);
     DEBUG ((DEBUG_INFO, "Boot Guard Boot Status = %llx\n", BootGuardBootStatus));
 
+    DEBUG ((DEBUG_INFO, "HeciBaseAddress = %llx\n", HeciBaseAddress));
     ///
     /// Read ME FWS Registers
     ///
-    MeFwSts4 = MmioRead32 (HeciBaseAddress + R_ME_HFS_4);
-    DEBUG ((DEBUG_INFO, "ME FW STS 4 = %x\n", MeFwSts4));
+    if (HeciBaseAddress != 0) {
+      MeFwSts4 = MmioRead32 (HeciBaseAddress + R_ME_HFS_4);
```

Comment:
Would you add comments when HeciBaseAddress is expected to 0x0?

Reply:
done